cmd/root.go

```diff
@@ -13,5 +19,47 @@ var rootCmd = &cobra.Command{
 }
 
 func Execute() {
-	rootCmd.Execute()
+	if err := rootCmd.Execute(); err != nil {
+		log.Fatal(err)
+		os.Exit(-1)
+	}
+}
+
+func init() {
+	cobra.OnInitialize(initConfig)
+}
+
+func initConfig() {
+	// set up ENV var support
+	viper.SetEnvPrefix("pflt")
+	viper.AutomaticEnv()
+
+	// set up optional config file support
+	viper.SetConfigName("config")
+	viper.SetConfigType("yaml")
```

Comment:
1. Do we have a use case that demands a configuration file?
2. If not, can we proceed without enabling config-file use cases? cc @bcrochet as I know you mentioned that a set-and-forget approach might be valuable
3. Do we prefer a configuration file to CLI flags?